cloudify_cli/commands/executions.py

```diff
@@ -189,11 +192,8 @@ def manager_start(workflow_id,
     events_message = "* Run 'cfy events list -e {0}' to retrieve the " \
                      "execution's events/logs"
     original_timeout = timeout
-    logger.info('Executing workflow {0} on deployment {1} '
-                '[timeout={2} seconds]'.format(
-                    workflow_id,
-                    deployment_id,
-                    timeout))
+    log_msg = 'Executing workflow `{0}` on deployment `{1}` [timeout={2}' \
+              ' seconds]'.format(workflow_id, deployment_id, timeout)
```

Comment:
i think let's still log BEFORE actually running the execution... and then another log if queued?

Reply:
Yes, makes sense